assets/js/vue/google-map-entry.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  tagName: 'google-map-entry',
+  computed: {
+    classList() {
+      return ['media vue-component', this.entry.icon === 'phone-charger' ? '' : 'mb-5'];
+    },
+  },
+  props: {
+    entry: {
```

Comment:
Kannst du Entry in atomare Properties ändern, Icon + Headline? Dann weiß man auhc was man erwartet in der Komponente.